cmake/modules/FindGLEW.cmake

```diff
@@ -62,6 +62,12 @@ if (WIN32)
         PATH_SUFFIXES
             Release/${ARCH}
         DOC "The GLEW library")
+
+    if ("${GLEW_LIBRARY}" MATCHES "glew32s(d|)")
+        add_definitions("/DGLEW_STATIC")
```

Comment:
I believe this should be GLEW_BUILD (not GLEW_STATIC) -- I will confirm and update the PR.